packages/cli-main/src/index.ts

```diff
@@ -1,8 +1,9 @@
 import {runCLI} from '@shopify/cli-kit/node/cli'
 
-async function runShopifyCLI() {
+async function runShopifyCLI(development: boolean) {
```

Comment:
What about using an object for the parameter instead so that you can see what the parameter means? That way we could use `runCLI({development: true})` instead of `runCLI(true)`. And maybe a default value of false?

Reply:
Much better. I'll update it.